.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   extends: '@mate-academy/eslint-config',
-  parser: "babel-eslint",
```

Comment:
The parser option is removed, which might cause issues if your project uses Babel-specific syntax. If you still need it, add it back: parser: 'babel-eslint',